docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1590,6 +1590,10 @@ In the `ibmcom/ibm-fhir-server` docker image, these logs will be at `/logs/joblo
 
 Note, if you are using the default derby, the logs are overwritten upon restart of the server. You should use Db2 or Postgres for production purposes.
 
+### 4.10.6 Known Limitations
+
+The IBM FHIR Server's fhir-bulkdata-webapp does not support [persistence interceptors](https://github.com/IBM/FHIR/blob/main/docs/src/pages/guides/FHIRServerUsersGuide.md#43-persistence-interceptors). As the IBM FHIR Server's notifications feature is implemented as a persistence interceptor, the afterCreate and afterUpdate operation are not executed. 
```

Comment:
great idea to add this.  minor suggested edits to the specific content:
```suggestion
The IBM FHIR Server's fhir-bulkdata-webapp does not support [persistence interceptors](https://github.com/IBM/FHIR/blob/main/docs/src/pages/guides/FHIRServerUsersGuide.md#43-persistence-interceptors). Therefor, $import requests will not lead to `beforeCreate`/`beforeUpdate` or `afterCreate`/`afterUpdate` method calls and $export requests will not lead to `beforeRead`/`afterRead` or `beforeSearch`/`afterSearch` method calls.
Because the IBM FHIR Server's notifications feature is implemented as a persistence interceptor, bulk operations will not result in any notification events.
```